app/components/HeaderNotifications/index.js

```diff
@@ -115,11 +123,6 @@ export default class NotificationsDropdown extends Component<Props, State> {
             <div style={{ maxHeight: '300px', overflowY: 'overlay' }}>
               {this.renderNotifications(notifications)}
             </div>
-            <div className={styles.seeAllWrapper}>
-              <Link onClick={this.seeAll} className={styles.seeAll}>
```

Comment:
Should also remove the `seeAll` function and styles from the CSS file, right?

Reply:
I'll do that :ok_hand: